### DISCUSSION TITLE
GUI with D3 Plot Conversion option

### DISCUSSION THREAD

**Discussion by okkarkyaw122** [OpenRadioss/Tools#30] [Q&A] [answered]
Hello, I want to use GUI with an option that includes Anmi-D3plot conversion. But Whenever I opened 
 "f2848fd", it doesn't show Anmi to D3plot conversion option. 
 
 Fig 1 I want a Gui like this.
![openradioss_gui](https://github.com/user-attachments/assets/9eb6110e-a3ea-4291-b0c3-b36f68efa432)

This is whats happening in my GUI
![image](https://github.com/user-attachments/assets/14a3a35c-2173-4c26-85de-1b90dc658cc6)

**Comment by okkarkyaw122** [comment]
BTW, I am aware that "*D3plot button appears only if Vortex-CAE D3plot converter is detected." I already installed Vortex with 

python -m pip install lasso-python

this script. But it still cannot detect the converter. Are there any more steps?

**Comment by PaulAltair** [comment, marked as answer]
lasso-python is just a dependency for Vortex-Radioss, it is not the Vortex-Radioss tool itself

you need to install Vortex-Radioss from [Vortex-Radioss](https://github.com/Vortex-CAE/Vortex-Radioss)

(the vortex_radioss folder needs to be in your python installation 'Lib' folder)

**Reply by okkarkyaw122** [reply]
Hello. Thank you for your instruction. It is now operational.

**Reply by okkarkyaw122** [reply]
I have zero experience in coding. However, I would love to share how I was able to achieve that. Hope this helps other people too.
After copying the vortex_radioss folder, I run this script in Common Prompt

from vortex_radioss.animtod3plot.RadiossReader import RadiossReader
help(RadiossReader)

If you can run the above command without any import errors and see the help documentation for RadiossReader, then you have successfully installed and configured Vortex-Radioss.
